src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -47,7 +47,7 @@
   private String url;
   private String email;
   // In method setModeratingOrgs(), ArrayList will be set with all Organizations that this GivrUser moderates.
-  private ArrayList<Entity> moderatingOrgs = new ArrayList<Entity>();
+  public ArrayList<Entity> moderatingOrgs = new ArrayList<Entity>();
```

Comment:
why is this public? shouldn't it be private and have a get method?

Reply:
In the test cases I'm assigning a hardcoded array directly to this variable since "setModeratingOrgs" is already implemented in this class. I figured it would be confusing to create another "setModeratingOrgs" function to accomodate for this task the test case need to do